### PR TITLE
Added Multi-threading

### PR DESCRIPTION
**Pull request by shivankar-madaan**
Need help with naming the function
Currently given as new_func,which is not appropriate.
Open to any suggestion

### CODE REVIEW

File: modules/getweblinks.py [hunk 1]
```diff
@@ -84,6 +86,15 @@ def add_red(link):
     colors = Bcolors()
     return '\t' + colors.On_Red + link + colors.ENDC
 
+def new_func(link):
```
Comment (KingAkeem) [inline modules/getweblinks.py]: Rename the function to possibly `def get_link_status(link):`?

Comment (shivankar-madaan) [reply modules/getweblinks.py]: done

**Comment by PSNAppz** [comment]
@shivankar-madaan Status on this ?

**Comment by shivankar-madaan** [comment]
Will be working on this .. from the start of next week

**Comment by PSNAppz** [comment]
@shivankar-madaan Ok good

**Comment by PSNAppz** [comment]
@shivankar-madaan Status?

**Comment by PSNAppz** [comment]
@shivankar-madaan Status on this ? Should we wait or close this?

**Comment by KingAkeem** [comment]
If we don't get a response within a week, we should just close this pr.

**Comment by iamPrakhar** [comment]
Hi , I can take this up.

**Comment by PSNAppz** [comment]
@iamPrakhar Great!. You can start working on the same branch I guess.

**Comment by KingAkeem** [comment]
@shivankar-madaan Conflicts need to be resolved.

**Comment by iamPrakhar** [comment]
Its better that i will work on the latest branch.

**Comment by KingAkeem** [comment]
@iamPrakhar Ok, sounds good to me! Just making sure that everyone is aware.

**Comment by PSNAppz** [comment]
@iamPrakhar I need a deadline on this.

**Comment by iamPrakhar** [comment]
@PSNAppz i am working on this , will provide you an update by tomorrow EOD.

**Comment by PSNAppz** [comment]
@iamPrakhar Please comment your email id. I will invite you to our slack team.

**Comment by iamPrakhar** [comment]
@PSNAppz  prakhar.rulingking@gmail.com

**Comment by PSNAppz** [comment]
@iamPrakhar Please check your mail.

**Comment by KingAkeem** [comment]
This feature has been added.

**Pull request closed without merging**